app/utils/JSONSchemaValidator.scala

```diff
@@ -36,9 +36,9 @@ class JSONSchemaValidator {
   val ssfnRequestSchema = "/schemas/statement-search-failure-notification-request-schema.json"
   val ssfnErrorResponseSchema = "/schemas/statement-search-failure-notification-error-response-schema.json"
   val ssfnSecureMessageRequestSchema = "/schemas/secure-message-request-schema.json"
-  val acc44RequestSchema = "/schemas/acc44-fe-request-schema-v1.0.json"
-  val acc44ResponseSchema = "/schemas/acc44-fe-response-schema-success-v1.0.json"
-  val acc45RequestSchema = "/schemas/acc45-fe-request-schema-v1.0.json"
+  val acc44RequestSchema = "/schemas/acc44-fe-request-schema-v1.2.json"
+  val acc44ResponseSchema = "/schemas/acc44-fe-response-schema-success-v1.2.json"
+  val acc45RequestSchema = "/schemas/acc45-fe-request-schema-v1.1.json"
```

Comment:
Up to you - these values could go in a companion object to be imported rather than injected.

Reply:
as discussed over the call, we can take this as part of separate tech debt task as this file is being used at quite a few places